sourcepoint-ui/TCFv2_banner_amp.css

```diff
@@ -123,6 +123,7 @@ https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
 
 .gu-content {
     border-top: 1px solid rgb(80, 105, 145) !important;
+    max-height: calc(568px * 0.6) !important; /* Opera mini does not support viewport units. Calculates 60% of the height of an iPhone SE viewport */
```

Comment:
I think you can do the calculation yourself and get rid of `calc`: `340px`.
Is this iPhone SE referring to 1<sup>st</sup> gen?